crates/analytics/src/disputes/accumulators.rs

```diff
@@ -17,7 +17,7 @@ pub struct RateAccumulator {
 }
 #[derive(Debug, Default)]
 #[repr(transparent)]
-pub struct SumAccumulator {
+pub struct PaymentProcessedAmountAccumulator {
```

Comment:
We can name it as `DisputedAmountAccumulator`

Reply:
done